pages/partials/assessmentScorePanel.ejs

```diff
@@ -6,6 +6,11 @@
     <div class="d-flex justify-content-center">
       <a class="btn btn-info" href="<%= urlPrefix %>/assessment_instance/<%= assessment_instance.id %>/">Assessment overview</a>
     </div>
+    <% if (locals.workspaceUrl.split('/').pop() !== 'null') { %>
+    <div class="d-flex justify-content-center mt-2">
+      <a class="btn btn-info" target="_blank" href="<%= locals.workspaceUrl %>">Open workspace</a>
+    </div>
+    <% } %>
```

Comment:
Can you explain why you added this? I was thinking the link would be provided solely by the element.

Reply:
> 
> Can you explain why you added this? I was thinking the link would be provided solely by the element.

Yeah, I was planning to demo/pitch this at the meeting tomorrow, but I forgot you won't be able to make it. I'll try to summarize my thinking:

I like how `pl-workspace` allows instructors to place the button within the flow of a question, but I think it's also good to provide an additional predictable/reliable version of the button.

Say I'm coming back to work on a question, or I'm an instructor wanting to test the workspace, or some kind of analogous situation where I don't need/want to go through the question narrative -- it'd be preferable not having to scroll through the question every time to find `pl-workspace` again, so I thought it might be good UX to also include a standard sidebar button.

Reply:
While I agree with the motivation for a different way to access workspaces, I don't think putting this type of button in the sidebar is "standard". I think keeping it somewhere in the normal question flow is more desirable. A few options that might be better:

* The question header
* The question footer
* A dedicated panel

I mocked up the question header option; what do you think about this?

<img width="525" alt="question panel with open workspaces button in the header" src="https://user-images.githubusercontent.com/1476544/86680308-47a0e780-bfb3-11ea-9603-f5d1bf9e8f8c.png">

Reply:
I like it!